hibike/devices/BatteryBuzzer/battery_buzzer.cpp

```diff
@@ -38,7 +38,7 @@ void setup() {
 
   delay(100);  // Wait for the first reading to finish (50ms minimum)
 
-  hibike_setup();
+  hibike_setup(500); //parameter indicateds disable latency in milliseconds
```

Comment:
"indicateds"?  "Indicates"?  Seems like copy-pasta has spread this spelling mistake around.